### ISSUE TITLE
[JDK11] TableView checkbox column reflection problem

### ISSUE THREAD

**Issue by pavlus** [edvin/tornadofx#1217]
**TornadoFX 2.0.0-SNAPSHOT**

Code like 
```
tableview {
    column("Something",Model::booleanProperty)
        .useCheckbox(/*'editable' value doesn't matter*/)
}
```
throws cross-module reflection access exception outside of user code, but inside TornadoFX code, when this table-view is displayed:
```
00:17:16.853 [JavaFX Application Thread] ERROR ErrorHandler - Uncaught error
java.lang.reflect.InaccessibleObjectException: Unable to make protected javafx.collections.ObservableList javafx.scene.Parent.getChildren() accessible: module javafx.graphics does not "opens javafx.scene" to unnamed module @4ebf6981
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:340)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:280)
	at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:198)
	at java.base/java.lang.reflect.Method.setAccessible(Method.java:192)
	at tornadofx.FXKt.getChildrenReflectively(FX.kt:684)
	at tornadofx.FXKt.getChildList(FX.kt:675)
	at tornadofx.FXKt.addChildIfPossible(FX.kt:613)
	at tornadofx.FXKt.addChildIfPossible$default(FX.kt:499)
	at tornadofx.ControlsKt.checkbox(Controls.kt:881)
	at tornadofx.ControlsKt.checkbox$default(Controls.kt:159)
	at tornadofx.ItemControlsKt$useCheckbox$$inlined$apply$lambda$1$1.invoke(ItemControls.kt:473)
	at tornadofx.ItemControlsKt$useCheckbox$$inlined$apply$lambda$1$1.invoke(ItemControls.kt)
	at tornadofx.NodesKt.cache(Nodes.kt:1286)
	at tornadofx.NodesKt.cache$default(Nodes.kt:1285)
	at tornadofx.ItemControlsKt$useCheckbox$$inlined$apply$lambda$1.invoke(ItemControls.kt:471)
	at tornadofx.ItemControlsKt$useCheckbox$$inlined$apply$lambda$1.invoke(ItemControls.kt)
	at tornadofx.SmartTableCell.updateItem(TableView.kt:96)
	at javafx.controls/javafx.scene.control.TableCell.updateItem(TableCell.java:665)
	at javafx.controls/javafx.scene.control.TableCell.indexChanged(TableCell.java:469)
	at javafx.controls/javafx.scene.control.IndexedCell.updateIndex(IndexedCell.java:120)
	at javafx.controls/javafx.scene.control.skin.TableRowSkinBase.updateCells(TableRowSkinBase.java:539)
	at javafx.controls/javafx.scene.control.skin.TableRowSkinBase.<init>(TableRowSkinBase.java:159)
	at javafx.controls/javafx.scene.control.skin.TableRowSkin.<init>(TableRowSkin.java:89)
	at javafx.controls/javafx.scene.control.TableRow.createDefaultSkin(TableRow.java:213)
	at javafx.controls/javafx.scene.control.Control.doProcessCSS(Control.java:897)
	at javafx.controls/javafx.scene.control.Control.access$000(Control.java:83)
	at javafx.controls/javafx.scene.control.Control$1.doProcessCSS(Control.java:89)
	at javafx.controls/com.sun.javafx.scene.control.ControlHelper.processCSSImpl(ControlHelper.java:67)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(NodeHelper.java:145)
	at javafx.graphics/javafx.scene.Parent.doProcessCSS(Parent.java:1397)
	at javafx.graphics/javafx.scene.Parent.access$400(Parent.java:79)
	at javafx.graphics/javafx.scene.Parent$1.doProcessCSS(Parent.java:125)
	at javafx.graphics/com.sun.javafx.scene.ParentHelper.processCSSImpl(ParentHelper.java:98)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(NodeHelper.java:145)
	at javafx.graphics/javafx.scene.Parent.doProcessCSS(Parent.java:1397)
	at javafx.graphics/javafx.scene.Parent.access$400(Parent.java:79)
	at javafx.graphics/javafx.scene.Parent$1.doProcessCSS(Parent.java:125)
	at javafx.graphics/com.sun.javafx.scene.ParentHelper.processCSSImpl(ParentHelper.java:98)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(NodeHelper.java:145)
	at javafx.graphics/javafx.scene.Parent.doProcessCSS(Parent.java:1397)
	at javafx.graphics/javafx.scene.Parent.access$400(Parent.java:79)
	at javafx.graphics/javafx.scene.Parent$1.doProcessCSS(Parent.java:125)
	at javafx.graphics/com.sun.javafx.scene.ParentHelper.processCSSImpl(ParentHelper.java:98)
	at javafx.controls/com.sun.javafx.scene.control.ControlHelper.superProcessCSSImpl(ControlHelper.java:63)
	at javafx.controls/com.sun.javafx.scene.control.ControlHelper.superProcessCSS(ControlHelper.java:55)
	at javafx.controls/javafx.scene.control.Control.doProcessCSS(Control.java:886)
	at javafx.controls/javafx.scene.control.Control.access$000(Control.java:83)
	at javafx.controls/javafx.scene.control.Control$1.doProcessCSS(Control.java:89)
	at javafx.controls/com.sun.javafx.scene.control.ControlHelper.processCSSImpl(ControlHelper.java:67)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(NodeHelper.java:145)
	at javafx.graphics/javafx.scene.Node.processCSS(Node.java:9529)
	at javafx.graphics/javafx.scene.Node.processCSS(Node.java:9522)
	at javafx.graphics/javafx.scene.Node.processCSS(Node.java:9522)
	at javafx.graphics/javafx.scene.Node.processCSS(Node.java:9522)
	at javafx.graphics/javafx.scene.Scene.doCSSPass(Scene.java:569)
	at javafx.graphics/javafx.scene.Scene.access$3400(Scene.java:172)
	at javafx.graphics/javafx.scene.Scene$ScenePulseListener.pulse(Scene.java:2477)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.lambda$runPulse$2(Toolkit.java:412)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.runPulse(Toolkit.java:411)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.firePulse(Toolkit.java:438)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:519)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:499)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulseFromQueue(QuantumToolkit.java:492)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$11(QuantumToolkit.java:320)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication.enterNestedEventLoopImpl(Native Method)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication._enterNestedEventLoop(GtkApplication.java:347)
	at javafx.graphics/com.sun.glass.ui.Application.enterNestedEventLoop(Application.java:509)
	at javafx.graphics/com.sun.glass.ui.EventLoop.enter(EventLoop.java:107)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.enterNestedEventLoop(QuantumToolkit.java:591)
	at javafx.graphics/javafx.stage.Stage.showAndWait(Stage.java:465)
	at javafx.controls/javafx.scene.control.HeavyweightDialog.showAndWait(HeavyweightDialog.java:162)
	at javafx.controls/javafx.scene.control.Dialog.showAndWait(Dialog.java:345)
	at tornadofx.DefaultErrorHandler.showErrorDialog(ErrorHandler.kt:122)
	at tornadofx.DefaultErrorHandler.access$showErrorDialog(ErrorHandler.kt:15)
	at tornadofx.DefaultErrorHandler$uncaughtException$1.run(ErrorHandler.kt:43)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:428)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:427)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

Additional info: Linux, OpenJFX 11, OpenJDK 11, Kotlin 1.3.71 (stdlib and reflect), shaded multiplatform jar.

**Comment by pavlus** [comment]
Seems like it was fixed by #964 but somehow I don't see it in sources, probably old snapshot.

**Comment by pavlus** [comment]
Ok, I see now: #1104 broke it again by reverting changes from #964
